includes/forms.inc

```diff
@@ -358,6 +358,38 @@ function islandora_ip_embargo_add_range_form($form, &$form_state, $list_identifi
 }
 
 /**
+ * Validate handler for adding an IP Range.
+ *
+ * @param array $form
+ *   The Drupal form definition.
+ * @param array $form_state
+ *   The Drupal form state.
+ */
+function islandora_ip_embargo_add_range_form_validate(array $form, array &$form_state) {
+  $comparable_low = ip2long($form_state['values']['low_end']);
+  $comparable_high = ip2long($form_state['values']['high_end']);
+
+  $error = FALSE;
+  if (empty($comparable_low)) {
+    $error = TRUE;
+    form_set_error('low_end', t('Low end IP is not valid.'));
+  }
+
+  if (empty($comparable_high)) {
+    $error = TRUE;
+    form_set_error('high_end', t('High end IP is not valid.'));
+  }
+
+  if ($error) {
+    form_set_error('', t('IP addresses must be in dot-decimal notation with no leading zeros.'));
```

Comment:
This doesn't need to be it's own error, it is extra information on the two previous errors.  It could also be misleading the way it is phrased right now; people might take it to mean that this must be the mistake they have made, when they could have made a different one.